src/in-memory-backend.service.ts

```diff
@@ -220,7 +220,7 @@ export class InMemoryBackendService {
       if ('commands' === reqInfo.base.toLowerCase()) {
         return this.commands(reqInfo);
 
-      } else if (this.inMemDbService[reqMethodName]) {
+      } else if (this.inMemDbService[reqMethodName] && reqInfo.collection) {
```

Comment:
No this this one. The override method should run whether `reqInfo.collection` is present or not.